sdk/apimanagement/arm-apimanagement/src/operations/apiOperation.ts

```diff
@@ -85,16 +90,23 @@ export class ApiOperationImpl implements ApiOperation {
     resourceGroupName: string,
     serviceName: string,
     apiId: string,
-    options?: ApiOperationListByApiOptionalParams
+    options?: ApiOperationListByApiOptionalParams,
+    settings?: PageSettings
   ): AsyncIterableIterator<OperationContract[]> {
-    let result = await this._listByApi(
-      resourceGroupName,
-      serviceName,
-      apiId,
-      options
-    );
-    yield result.value || [];
-    let continuationToken = result.nextLink;
+    let result: ApiOperationListByApiResponse;
+    let continuationToken = settings?.continuationToken;
```

Comment:
Not sure we could get continuaToken correctly if the settings is empty.
@qiaozha How do you think this code shippet?